tests/formdata.js

```diff
@@ -0,0 +1,26 @@
+var http = require('k6/http');
+var check = require('k6').check
+var FormData = require('../lib/formdata/0.0.1/index.js').FormData;
+
+var logo = open('./data/logo.png', 'b');
+
+exports.testPost = function() {
+  var fd = new FormData();
+  fd.append('binArray', http.file(logo, 'logo.png', 'image/png'));
+  fd.append('ArrayBuffer', http.file(new Uint8Array(logo).buffer, 'logo.png', 'image/png'));
```

Comment:
```suggestion
  fd.append('ArrayBuffer', { data:new Uint8Array(logo).buffer, filename: 'logo.png', content_type: 'image/png' });
```

This way we also will check that you can provide a simple Object with the correct keys and it will work. Also works around the current k6 not supporting ArrayBuffer ;)